TCMSApp/src/client/app/layout/sidebar.service.js

```diff
@@ -16,20 +16,37 @@
 
         // Gets proects from DB
         function findProjects() {
-
             var deferred = $q.defer();
+            var currentUser = user.id;
+            if (user.trelloUserID !== undefined)
+                currentUser = user.trelloUserID;
```

Comment:
what is the logic of this part?  is was better when `currentUser` was always our current user.